cassandra-4/src/main/java/io/debezium/connector/cassandra/Cassandra4CommitLogNearRealTimeParser.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.cassandra;
+
+import java.util.List;
+
+import org.apache.cassandra.db.commitlog.CommitLogPosition;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.debezium.connector.base.ChangeEventQueue;
+
+public class Cassandra4CommitLogNearRealTimeParser extends Cassandra4CommitLogParserBase {
```

Comment:
I dont like the name of this class, it is little bit .... verbose. Not sure what to call it like though.

Reply:
Dropped the word "near" from the name, hope it helps to shorten it.